tests/foreman/cli/test_oscap.py

```diff
@@ -572,16 +572,22 @@ def test_positive_associate_scap_policy_with_hostgroups(self, scap_content, modu
 
             1. Oscap should be enabled.
             2. Oscap-cli hammer plugin installed.
-            3. More than 1 hostgroups
+            3. More than 1 policies assigned to hostgroups
 
         :steps:
 
             1. Login to hammer shell.
             2. Execute "policy" command with "create" as sub-command.
             3. Pass valid parameters.
-            4. Associate multiple hostgroups with policy
+            4. Associate multiple policies with hostgroup
+            5. Delete hostgroup
 
         :expectedresults: The policy is created and associated successfully.
+            Policies can be listed after hostgroup removal.
+
+        :bz: 1728157
```

Comment:
```suggestion
        :bz: 1728157
        
        :Verifies: SAT-19502
```
@pondrejk As we're moving away from Bugzilla, could we also add Jira issue here?

Reply:
Added!